README.md

```diff
@@ -158,12 +158,13 @@ Maximum number of processes and open files.
 ### ./defaults/main/misc.yml
 
 ```yaml
+install_aide: yes
 reboot_ubuntu: false
 redhat_rpm_key:
   - 567E347AD0044ADE55BA8A5F199E2F91FD431D51
   - 47DB287789B21722B6D95DDE5326810137017186
 ```
-
+If `install_aide: true` then AIDE IDS will be installed and configured. If you plan use another IDS then you can disable AIDE installation completely here.<br>
```

Comment:
Just use a blank line instead of `<br>`.

Reply:
Fixed